stefaniaPopescu26/assignment3/WordSearch.java

```diff
@@ -21,11 +20,41 @@
 */
 
 public class WordSearch {
+    private static final int row_offset[] =
+            new int[]{-1, -1, -1, 0, 0, 1, 1, 1};
+    private static final int col_offset[] =
+            new int[]{-1, 0, 1, -1, 1, -1, 0, 1};
     private Dictionary dictionary;
     private String[][] grid;
     private int rows;
     private int cols;
 
+    class Pair {
```

Comment:
Nice! 
Maybe even mark row and col as final and allow only setting them in the constructor. You actually never use the setter methods